builders/interact/substrate-api/sidecar-api.md

```diff
@@ -0,0 +1,495 @@
+    |:--------------------:|:----------------------------------------------------------------------:|
+    |       Chain ID       |    `extrinsics[extrinsic_number].args.transaction.eip2930.chainId`     |
+    |        Nonce         |     `extrinsics[extrinsic_number].args.transaction.eip2930.nonce`      |
+    |      Gas price       |    `extrinsics[extrinsic_number].args.transaction.eip2930.gasPrice`    |
+    |      Gas limit       |    `extrinsics[extrinsic_number].args.transaction.eip2930.gasLimit`    |
+    |        Value         |     `extrinsics[extrinsic_number].args.transaction.eip2930.value`      |
+    |     Access list      |   `extrinsics[extrinsic_number].args.transaction.eip2930.accessList`   |
+    |      Signature       | `extrinsics[extrinsic_number].args.transaction.eip2930.oddYParity/r/s` |
+    |    Sender address    |      `extrinsics[extrinsic_number].events[event_number].data[0]`       |
+    |  Recipient address   |      `extrinsics[extrinsic_number].events[event_number].data[1]`       |
+    |       EVM hash       |      `extrinsics[extrinsic_number].events[event_number].data[2]`       |
+    | EVM execution status |      `extrinsics[extrinsic_number].events[event_number].data[3]`       |
+
+!!! note
+    For Substrate transactions, the "Nonce" and "Signature" fields are under `extrinsics[extrinsic_number]`. For EVM transactions, the "Nonce" and "Signature" fields are under `extrinsics[extrinsic_number].args.transaction[transaction_type]`, leaving the "Nonce" and "Signature" under `extrinsics[extrinsic_number]` to be `null`. 
+
+    A successfully executed EVM transaction will return either `succeed: "Stopped"` or `succeed: "Returned"` under the "EVM Execution Status" field.
+
+## Monitor Token Balance Transfers {: #monitor-transfers }
+
+The following code samples will demonstrate how to listen to both native token transfers, sent via Substrate or Ethereum API, and ERC-20 token transfers sent via the Ethereum API, using Substrate API Sidecar. Transfers via the Ethereum API are only applicable to EVM ContainerChains.
+
+### Native Token Transfers { #native-token-transfers }
+
+Both Tanssi, non-EVM ContainerChains and EVM ContainerChains can perform Substrate-based native token balance transfers.
+
+The following code snippet uses the Axios HTTP client to query the Sidecar endpoint [`/blocks/head`](https://paritytech.github.io/substrate-api-sidecar/dist/){target=_blank} for the latest finalized block, and then decodes the block for the `from`, `to`, `value`, `tx hash` and `transaction status` of native token transfers at both the EVM and Substrate API level. 
+
+```typescript
+--8<-- 'code/substrate-api/sidecar-transfer.ts'
+```
+
+### ERC-20 Token Transfers {: #erc-20-token-transfers }
+
+Events emitted by smart contracts such as an ERC-20 token contract deployed on EVM ContainerChains can be decoded from Sidecar block JSON objects. The nesting structure is as following:
+
+```JSON
```

Comment:
```suggestion
```
```

Reply:
I'd delete the JSON type of code. It renders the text in a funny way.
(Applies to every tree type structure description in the doc)